libs/application/templates/children-residence-change/src/fields/Confirmation/index.tsx

```diff
@@ -1,27 +1,78 @@
 import React from 'react'
 import { useIntl } from 'react-intl'
-import { Box } from '@island.is/island-ui/core'
+import { Box, Text } from '@island.is/island-ui/core'
 import { confirmation, copyUrl } from '../../lib/messages'
-import { DescriptionText, CopyUrl } from '../components'
+import { CopyUrl, DescriptionText } from '../components'
+import { CRCFieldBaseProps } from '../..'
+import ContractAccordionOverview from '../components/ContractOverviewAccordion/ContractOverviewAccordion'
 
-const Confirmation = () => {
+const Confirmation = ({ application }: CRCFieldBaseProps) => {
   const { formatMessage } = useIntl()
+  const { answers, externalData } = application
+
   return (
     <>
       <Box marginTop={3}>
         <DescriptionText
           text={confirmation.general.description}
-          format={{ applicationNumber: '12345' }}
+          format={{
+            emailParagraph: answers.counterParty.email
+              ? formatMessage(
+                  confirmation.general.description.paragraphs.email,
+                  { email: answers.counterParty.email },
+                )
+              : '',
+            phoneNumberParagraph: answers.counterParty.phoneNumber
+              ? formatMessage(
+                  confirmation.general.description.paragraphs.phoneNumber,
+                  { phoneNumber: answers.counterParty.phoneNumber },
+                )
+              : '',
+            count: [
+              answers.counterParty.email,
+              answers.counterParty.phoneNumber,
+            ].reduce((value, item) => {
+              return item ? value + 1 : value
+            }, 0),
+          }}
         />
       </Box>
-      <Box marginTop={5}>
+      <Text variant="h4" marginTop={3}>
+        {formatMessage(confirmation.nextSteps.title)}
+      </Text>
+      <Box marginTop={2}>
+        <DescriptionText
+          text={confirmation.nextSteps.description}
+          format={{
+            parentBName:
+              externalData.nationalRegistry.data.children[0].otherParent
+                .fullName,
+          }}
+        />
+      </Box>
+      <Box marginTop={4}>
         <CopyUrl
           title={formatMessage(copyUrl.title)}
           inputLabel={formatMessage(copyUrl.inputLabel)}
           buttonLabel={formatMessage(copyUrl.buttonLabel)}
           successMessage={formatMessage(copyUrl.successMessage)}
         />
       </Box>
+      <Box marginTop={3}>
+        <ContractAccordionOverview
+          title={formatMessage(confirmation.contractOverview.accordionTitle)}
+          id="id_1"
+          application={application}
+        />
+      </Box>
+      <Box marginTop={5}>
+        <img
+          src={
+            'https://images.ctfassets.net/8k0h54kbe6bj/6UGl8bkfOwUDKYveXfKkh0/c09265b9301b0be52c678a7197a64154/crc-application-submitted.svg'
+          }
+          alt="Umsókn sent inn mynd"
```

Comment:
```suggestion
          alt=""
```

Since this is purely a decorative (eye-candy) illustration I suggest leaving the alt tag empty:
https://www.w3.org/WAI/tutorials/images/decorative/